linux-anvil-alma/Dockerfile

```diff
@@ -23,8 +23,11 @@ ADD qemu-s390x-static /usr/bin/qemu-s390x-static
 # bust the docker cache so that we always rerun the installs below
 ADD https://loripsum.net/api /opt/docker/etc/gibberish
 
-# Resolves a nasty NOKEY warning that appears when using yum.
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
+# Add the archived repo URL and fix RPM imports
+ADD centos7-x86_64-vault.repo /tmp/centos7-x86_64-vault.repo
+ADD centos7-altarch-vault.repo /tmp/centos7-altarch-vault.repo
+ADD scripts/fix_rpm /opt/docker/bin/fix_rpm
+RUN /opt/docker/bin/fix_rpm
```

Comment:
AlmaLinux 8 doesn't need these right?

Also think we still need to `NOKEY` fix

```suggestion
# Resolves a nasty NOKEY warning that appears when using yum.
RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux
```

Reply:
It doesn't need the `centos7*` files, yes.
All `rpm --import`s are now in `fix_rpm`, including that one.

Reply:
Let's do some cleanup or possibly other/different cleanups after we've built some working cos7 containers.

Reply:
Let's not change images we don't need to change. This is focused on CentOS 7. Let's limit changes to that

Reply:
Don't really care.